.travis.yml

```diff
@@ -4,6 +4,13 @@ rvm:
 
 cache: bundler
 
+env:
+  global:
+  - CF_API="https://api.cloud.service.gov.uk"
+  - CF_ORG="openregister"
+  - CF_SPACE="sandbox"
```

Comment:
hmmm I guess this is fine for now but we need a way to deploy from the sandbox to production spaces

Reply:
Yes, I think there is a bigger question about promotion between stages, at that point we might want to look at something like CodePipeline.